src/esportsbot/cogs/MusicCog.py

```diff
@@ -712,15 +714,23 @@ def __format_api_data(self, data: list) -> List[dict]:
         for item in data:
             snippet = item.get("snippet")
             info = {"title": snippet.get("title", "Unable to get title, this is a bug"),
-                    # The url can be in different places depending on if the data is coming from a playlist or a video
-                    "link": snippet.get("resourceId", {}).get("videoId", item.get("id", "Unable to get link, this "
-                                                                                        "is a bug")),
-                    "thumbnail": snippet.get("thumbnails", {}).get("maxres", {}).get("url", "Unable to get thumbnail, "
+                    "thumbnail": snippet.get("thumbnails", {}).get("maxres", {}).get("url", "Unable to get thumbnail "
                                                                                             "this is a bug")}
```

Comment:
missing string concatenation operator
```suggestion
                                                                                            + "this is a bug")}
```

Reply:
The + isn't actually needed apparently as the splitting of the lines was done by Pycharm to keep within some PEP line length.